rosdep/python.yaml

```diff
@@ -6012,10 +6012,20 @@ python3-flask:
   fedora: [python3-flask]
   gentoo: [dev-python/flask]
   ubuntu: [python3-flask]
+python3-flask-cors:
+  debian: [python3-flask-cors]
+  ubuntu: [python3-flask-cors]
 python3-flask-restplus-pip:
   ubuntu:
     pip:
       packages: [flask-restplus]
+python3-flask-socketio:
+  debian:
+    pip:
+      packages: [flask-socketio]
```

Comment:
The debian version has 5.0 already: https://packages.debian.org/bullseye/python3-flask-socketio